apps/filecoin-site/src/app/learn/data/filecoinParticipants.ts

```diff
@@ -1,21 +1,19 @@
 import { PATHS } from '@/constants/paths'
 
-type FilecoinRole = {
-  title: string
-  description: string
-  cta: {
-    label: string
-    href: string
-  }
-}
+import type { SimpleCardProps } from '@/components/SimpleCard'
+
+export type FilecoinRole = Pick<
+  SimpleCardProps,
+  'title' | 'description' | 'cta'
+>
 
 export const filecoinParticipants: FilecoinRole[] = [
```

Comment:
```suggestion
export const filecoinParticipants: Array<FilecoinRole> = [
```

Also, it's a detail, but `as const` (below) has no effect if the variable is typed here.

`as const satisfies Array<FilecoinRole>` would have the intended effect.

But typing the variable here is also totally fine!

Reply:
From the TS master: https://www.youtube.com/watch?v=6M9aZzm-kEc